autoload/ember_tools/gf.vim

```diff
@@ -120,26 +120,51 @@ function! ember_tools#gf#Model()
   return ember_tools#ExistingLogicFile('app/models/'.dasherized_name)
 endfunction
 
-function! ember_tools#gf#TemplateComponent()
+"Should be a 'dasherized' name: `foo-bar/bazz`
+function! ember_tools#gf#FindComponent(component_name)
+  let component_file = s:FindComponentTemplate(a:component_name)
+  if component_file == ''
+    let component_file = s:FindComponentLogic(a:component_name)
+  endif
+  if component_file == ''
+    echomsg "Can't find component: ".a:component_name
+    return ''
+  endif
+  return component_file
+endfunction
+
+function! ember_tools#gf#AngleBracketTemplateComponent()
   if !ember_tools#IsTemplateFiletype()
     return ''
   endif
 
-  if !ember_tools#search#UnderCursor('^\s*\%(=\|{{#\|{{\)\{}\s*\zs\k\+')
+  if !ember_tools#search#UnderCursor('^\s*\%(=\|<\|<\/\)\{}\s*\zs\k\+')
     return ''
   endif
 
-  let component_name = expand('<cword>')
-  let component_file = s:FindComponentTemplate(component_name)
-  if component_file == ''
-    let component_file = s:FindComponentLogic(component_name)
+  set iskeyword+=:
+  set iskeyword-=/
+  let angle_bracketed_component_name = expand('<cword>')
+  set iskeyword-=:
+  set iskeyword+=/
```

Comment:
I'd like to be more precise in changing the `iskeyword` param -- what worries me in this case is that we're adding and removing properties, so if `:` is in `iskeyword` due to a user setting, it would be removed after this line. I'd like to try this:

``` vim
try
  let saved_iskeyword = &iskeyword
  set iskeyword += ...
  let angle_bracketed_...
finally
  let &iskeyword = saved_iskeyword
endtry
```

I see that in the codebase I also do what you did, so it's understandable you used the same pattern -- after the PR is merged, I'll go around and I'll apply it elsewhere too.